lib/src/_transport.dart

```diff
@@ -63,8 +63,11 @@ class Response {
   Map<String, dynamic>? _bodyAsMap;
   List? _bodyAsList;
 
+  /// The HTTP headers of the response.
+  final Map<String, String> headers;
```

Comment:
Do we have other headers that would be useful? If not, maybe we could just expose an optional `final String? warning;` field here, with `{this.warning}` in the constructor to make it backwards-compatible.

Reply:
I think your suggestion is better, thus I changed it.